recipes-core/images/avnet-image-full.inc

```diff
@@ -115,14 +120,9 @@ IMAGE_INSTALL:append:minized-sbc = "\
         wpa-supplicant \
 "
 
-IMAGE_INSTALL:append:mz = "\
-"
-
 IMAGE_INSTALL:append:pz = "\
-        haveged \
```

Comment:
Isn't haveged needed for encrypted connections like ssl and ssh?  Why remove it from avnet-image-full?